class NotSuccessfulResponseException(Exception):
    """
    Исключение, которое выбрасывается при получении неуспешного HTTP-ответа от внешнего сервиса.

    Это исключение сигнализирует о том, что запрос к API или другому сервису завершился с кодом ответа,
    отличным от ожидаемого, что указывает на возникновение ошибки при выполнении запроса.
    """
    pass
